### DISCUSSION TITLE
Nuxt is removing element from my header in prod, but not on localhost

### DISCUSSION THREAD

**Discussion by Knapstad** [nuxt/nuxt#22987] [Questions] [answered]
Hi.

I cant seem to find out why this is happening.
It's the usual it works on my machine.

I'm rewriting a site from vue too nuxt3 and in that process I've now deployed the app to netlify. and here the problem starts.
I'm using a default template that looks like this:
``` 
<template>
    <div>
        <TheHeader />
        <slot />
        <TheFooter />
    </div>
</template>
<script setup>

</script>
```
its called default.vue and is in a folder called layouts.
the header component looks like this:
``` 
<template>
  <header id="header">

    <NuxtLink to="/"><img class="logo" src="/logo.png" /></NuxtLink>
    <NuxtLink to="/">
      <p class="tagline">
      <h1>Bilabonnement</h1>
      - vi samler bilabonnement på en side</p>
    </NuxtLink>
  </header>
</template>
<style scoped>
a {
  text-decoration: none;
}

a:active {
  text-decoration: none;
}
</style>
```
And here is my folder structure:
```
components/
├─ TheFooter.vue
├─ TheHeader.vue
layouts/
├─ default.vue
nuxtbilabo/
pages/
plugins/
server/
app.vue
error.vue
nuxt.config.ts
```

the problem I am facing is that when deployed to prod, the tagline "Bilabonnement
      - vi samler bilabonnement på en side" is removed from the component after half a second. 
      
at first it renders but then it is removed. 

you can see for your self at this link [incandescent-sfogliatella-2d46fc.netlify.app](https://incandescent-sfogliatella-2d46fc.netlify.app)

Does any one know what I've doe wrong?

**Comment by TheAlexLichter** [comment, marked as answer]
Most likely because it is invalid HTML. `p` is not allowed in `h1`. Check out https://github.com/nuxt-modules/html-validator + semantic HTML to avoid such scenarios.

**Reply by Knapstad** [reply]
Thank you, I feel like an idiot :)
